cpp/src/common/cumlHandle.cpp

```diff
@@ -237,8 +242,18 @@ cudaStream_t cumlHandle_impl::getInternalStream(int sid) const {
   return _streams[sid];
 }
 
+std::vector<cudaStream_t> cumlHandle_impl::getInternalStreams() const {
```

Comment:
My proposed interface change above would enable to avoid the construction of a temporary `std::vector`.